.github/workflows/golangci-lint.yml

```diff
@@ -25,10 +25,14 @@ jobs:
           go-version-file: .go-version
 
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@2226d7cb06a077cd73e56eedd38eecad18e5d837  # v6.5.0
+        env:
+          CGO_ENABLED: 1
```

Comment:
Why is `CGO_ENABLED=1` added with this change?

Reply:
Maybe because I copied it from Beats :thinking:. I'll remove it.